dist/screeps.d.ts

```diff
@@ -1017,8 +1017,7 @@ interface SignDefinition {
 }
 interface StoreDefinition {
     [resource: string]: number;
-    energy?: number;
-    power?: number;
+    energy: number;
```

Comment:
```interface StoreDefinition {
    [key: string]: number;
    energy: number;
}

let x: Foo = { energy: 5 };

let y = x["bad"];

console.log(y);
```

logs `undefined`, but `y` is type `number` at compile time, meaning I can pass it into other functions that just take a `number` and I won't get any errors about passing `undefined` somewhere I shouldn't.

The correct typing for the indexer is `number | undefined` and then yes, I remember `energy` as always being 0, so leaving that in seems good.